app/controllers/api/v1/cookie_and_jwt_api/protocol_subscriptions_controller.rb

```diff
@@ -11,6 +11,20 @@ def show
           render json: @protocol_subscription, serializer: Api::ProtocolSubscriptionSerializer
         end
 
+        # Specify either a start_date or a start_time. A given start_date takes precedence.
+        # If a start_time is specified (and no start_date is specified),
+        # the start_date is chosen as the given start_time added to midnight the next day.
+        def create
+          result = SubscribeToProtocol.run!(
```

Comment:
it would be very nice to also do this async tbh, especially since you don't have to return anything, just the 204. We could add a subclass to activeinteraction with a `run_async` functin or soething, that automatically runs the run! in a background job.

Reply:
Alright, I'll try it.

Reply:
Ok I actually do a request for the status of the protocol subscription right after making one, so it's important that this happens synchronously. I'll leave this for now